.dependabot/config.yml

```diff
@@ -0,0 +1,55 @@
+# https://dependabot.com/docs/config-file/
+# validate with https://dependabot.com/docs/config-file/validator/
+version: 1
+update_configs:
+  - package_manager: 'javascript'
+    directory: '/'
+    update_schedule: 'weekly'
+    version_requirement_updates: 'auto'
+    default_labels:
+      - 'dependencies'
+      - 'worktree'
+    ignored_updates:
+      - match:
+          # These should be grouped. Maintainer advise: Watch https://github.com/babel/babel
+          # for new releases and file a PR if you got time.
+          dependency_name: '@babel/*'
+      - match:
+          # These should be grouped. Maintainer advise: Watch https://github.com/emotion-js/emotion
+          # for new releases and file a PR if you got time.
+          dependency_name: '@emotion/*'
+      - match:
+          # https://github.com/mui-org/material-ui/pull/17604#issuecomment-536262291
+          dependency_name: 'core-js'
+      - match:
+          # breaks /api routes
+          # can be upgraded once https://github.com/zeit/next.js/issues/8123 is resolved
+          dependency_name: 'next'
+          version_requirement: '9.x'
+      - match:
+          # 1.18 started adding trailing commas only compatible with typescript ^3.4
+          dependency_name: 'prettier'
+      - match:
+          # 2.0 started using ES modules instead of CommonJS modules
+          dependency_name: 'raw-loader'
+      # not ignoring `react`. We'll hijack those PRs and also upgrade the other
+      # packages in the facebook/react repository
```

Comment:
Ideally we would apply the same approach to every monorepo packages. Unfortunately `@babel` doesn't have a single package that always updates (`@babel/core` was not updated in 7.6.1).